### PR TITLE
USHIFT-618 Document NodePort port range extension limits

### PR DESCRIPTION
**Pull request by mangelajo**
Close-Issue: https://issues.redhat.com/browse/USHIFT-618

**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

File: docs/howto_config.md [hunk 1]
```diff
@@ -50,6 +50,38 @@ nodeName: ""
...
+  will be assigned randomly by the kube-apiserver.
+* Do not create any NodePort service for any system service port, MicroShift port,
+  or other services you expose on your device HostNetwork.
+
+List of ports that you must avoid:
+
+| Port          | Description                                                     |
+|---------------|-----------------------------------------------------------------|
+| 22/tcp        | SSH port
+| 1936/tcp      | Metrics service for the openshift-router, not exposed today
```
Comment (zshi-redhat) [inline docs/howto_config.md:74]: add 80 and 443 ports exposed for route deployment?

Comment (mangelajo) [reply docs/howto_config.md:74]: true! thanks I was looking at the netstat -nltp list which does not cover that.

Comment (mangelajo) [reply docs/howto_config.md:76]: Done

File: docs/howto_config.md [hunk 2]
```diff
@@ -50,6 +50,40 @@ nodeName: ""
...
+to NodePort Services. This option is useful when specific standard ports under the
+`30000-32767` need to be exposed. i.e., your device needs to expose the `1883/tcp`
+MQTT port on the network because some client devices cannot use a different port.
+
+If you use this option, you must be careful; NodePorts can overlap with system ports,
+causing malfunction of the system or MicroShift. Take the following considerations
+into account:
+
+* Do not create any NodePort service without an explicit `nodePort` selection. The port
+  will be assigned randomly by the kube-apiserver.
```
Comment (mangelajo) [inline docs/howto_config.md:65]:
```suggestion
  is be assigned randomly by the kube-apiserver.
```

Based on slack conversation with Shauna

**Pull request merged**